simplq/src/store/selectedQueue/selectedQueueSlice.js

```diff
@@ -29,6 +35,11 @@ const selectedQueueSlice = createSlice({
       state.status = action.payload.status;
       return state;
     },
+    // update queue settings
+    [updateQueueSettings.fulfilled]: (state, action) => {
+      state.maxQueueCapacity = action.payload.maxQueueCapacity;
```

Comment:
Let's set this only if `action.payload.maxQueueCapacity` is undefined?

Thinking out lot, just for extra safety?

Reply:
Do you mean update the state only if `state.maxQueueCapacity` is undefined OR don't update the state if `action.payload.maxQueueCapacity` is undefined?